### PR TITLE
Fix NaN in NodesGridDials percentage calculation

### PR DESCRIPTION
**Pull request by nLight**
Testing:
Have at least one (or many) app running with no Disk usage.

Before:
screen shot 2018-01-16 at 15 38 06

After:
jan-17-2018 11-33-48

Closes DCOS-20317

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [x] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by juliangieseke** [comment]
Alfred test this

**Comment by Poltergeist** [comment]
🎉  👯‍♂️👯 Alfred you can't test this. 👯 👯‍♂️ 🎉

**Comment by Poltergeist** [comment]
Alfred you can't test this.

**Comment by nLight** [comment]
Alfred test this

**Comment by DanielMSchmidt** [comment]
Alfred test this please :)

**Comment by orlandohohmeier** [comment]
Alfred could you please retest this PR?

**Comment by nLight** [comment]
Alfred test this

**Comment by juliangieseke** [comment]
Alfred test this

**Comment by Poltergeist** [comment]
run integration tests

**Comment by MatApple** [comment]
Heads up @nLight we have 4 test failures in integration https://cl.ly/0j0a3q2k113j/Image%202018-01-18%20at%2010.31.30%20AM.png

**Comment by nLight** [comment]
I see, I wonder if they are flaky. Waiting for video

**Comment by nLight** [comment]
run integration tests

File: plugins/nodes/src/js/components/NodesGridDials.js [hunk 1]
```diff
@@ -43,9 +43,11 @@ var NodesGridDials = React.createClass({
...
 
     Object.keys(resourcesByFramework).forEach(function(frameworkID) {
-      var percentage =
-        resourcesByFramework[frameworkID][props.selectedResource] * 100;
-      percentage /= node.getUsageStats(props.selectedResource).total;
+      const used =
+        resourcesByFramework[frameworkID][props.selectedResource] || 0;
+      const total = node.getUsageStats(props.selectedResource).total || 0;
+
+      const percentage = total === 0 ? 0 : used * 100 / total;
```
Comment (bstavroulakis) [inline plugins/nodes/src/js/components/NodesGridDials.js]: Doesn't this need a Math.round?

Comment (orlandohohmeier) [reply plugins/nodes/src/js/components/NodesGridDials.js]: We're rounding the data in https://github.com/dcos/dcos-ui/pull/2642/files/30b65a9a45a0e6029c6e6d9d779d7d26cefb8192#diff-aee5c9bb5a89d3f105fa091ea847c99bR101

**Pull request merged**